Gaijinizer patch/Map085.txt

```diff
@@ -316,27 +334,31 @@ that stage room. I usually lived there.#MTLed
 
 私だけ、施設のなかを自由に歩き回れた。
 > CONTEXT: Map085/events/5/pages/1/115/Dialogue/enemy6
-Other biological weapons in the same facility are
-I was locked in like a big coffin or showcase
-Only I was free to walk around in the facility.#MTLed
+Unlike the other biological weapons in the same
+facility who were locked away in coffins or
+display cases, I was free to walk around the
+facility.#Ron
 > END STRING
 
 > BEGIN STRING
 それどころか、欲しいものを言ったら
 
 無茶なもの以外は、大体用意してもらえたし。
 > CONTEXT: Map085/events/5/pages/1/120/Dialogue/enemy6
-On the contrary, if you say what you want
-Except for the innocence, they were mostly
-prepared.#MTLed
+In fact, if I asked for something they'd
+generally get it for me as long as it wasn't
+something too absurd.#Ron
```

Comment:
Minor flow:
>_**In fact... if I asked for something they'd
generally get it for me, as long as it wasn't
something too absurd.**_